apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -209,6 +221,30 @@ export const getCurrentAssessmentQuestions = (state) => {
   return currentScriptData[state.sectionAssessments.assessmentId];
 };
 
+// Returns the question text of the currently selected question in the current assessment.
+export const getCurrentQuestion = (state) => {
+  const isSurvey = isCurrentAssessmentSurvey(state);
+  if (isSurvey) {
+    const surveys = state.sectionAssessments.surveysByScript[state.scriptSelection.scriptId] || {};
+    const currentSurvey = surveys[state.sectionAssessments.assessmentId];
+    const questionData = currentSurvey.levelgroup_results;
+    if (questionData[state.sectionAssessments.questionIndex]) {
+      return questionData[state.sectionAssessments.questionIndex].question;
+    } else {
+      return '';
+    }
+  } else {
+    // Get question text for assessment.
+    const assessment = getCurrentAssessmentQuestions(state) || {};
```

Comment:
I'm not familiar with the syntax `|| {}`. How does it affect the assignment to assessment?

Reply:
It just means if the first expression is falsey (undefined), default `assessment` to the second expression. That way I can guarantee myself to execute the second line without error.